.cargo/config.toml

```diff
@@ -12,7 +12,74 @@ rustflags = [
 	# rustc additional warnings:
 	"--warn",
 	"unused_crate_dependencies",
-	# clippy additional warnings:
 	"--warn",
-	"clippy::wildcard_imports",
+	"rust_2018_idioms",
+	# clippy additional warnings:
+	# Suspicious
+	"-Wclippy::await_holding_lock",
+	# Restriction
```

Comment:
Nit: Other than readability, is there another purpose for grouping them by category here?
I wonder if it is easier to remove the categories, sorting everything alphabetically, to make it easier to find a rule/look up values in this file.

Reply:
Sounds good. Thanks!